MediaBrowser.Providers/Plugins/MusicBrainz/Plugin.cs

```diff
@@ -9,6 +9,18 @@ namespace MediaBrowser.Providers.Plugins.MusicBrainz
 {
     public class Plugin : BasePlugin<PluginConfiguration>, IHasWebPages
     {
+        public static Plugin Instance { get; private set; }
+
+        public override Guid Id => new Guid("8c95c4d2-e50c-4fb0-a4f3-6c06ff0f9a1a");
+
+        public override string Name => "MusicBrainz";
+
+        public override string Description => "Get artist and album metadata from any MusicBrainz server.";
+
+        public string DefaultServer = "https://musicbrainz.org";
```

Comment:
Why not make these defaults const?

Reply:
Add the suggestions and I can add them in batch from my phone.